src/component-library/TokenInput/TokenSelect.tsx

```diff
@@ -3,22 +3,35 @@ import { chain, mergeProps } from '@react-aria/utils';
 import { VisuallyHidden } from '@react-aria/visually-hidden';
 import { ChangeEventHandler, InputHTMLAttributes, useRef, useState } from 'react';
 
+import { CoinIcon } from '../CoinIcon';
+import { TokenStack } from '../TokenStack';
 import { assignFormRef, triggerChangeEvent } from '../utils/input';
-import { StyledChevronDown, StyledCoinIcon, StyledTicker, StyledTokenSelect } from './TokenInput.style';
+import { StyledChevronDown, StyledTicker, StyledTokenSelect } from './TokenInput.style';
 import { TokenData } from './TokenList';
 import { TokenListModal } from './TokenListModal';
 
+const Icon = ({ value, icons }: Pick<TokenSelectProps, 'value' | 'icons'>) => {
+  if (!value) return null;
+
+  if (icons?.length) {
+    return <TokenStack offset={icons.length > 2 ? '1/2' : '1/3'} tickers={icons} />;
```

Comment:
Should have picked this up in a previous review, but would it `1/2` and `1/3` are a bit inconsistent, for example with the `Spacing` type we use `half` and not `1/2`.